src/components/itemModal.html

```diff
@@ -53,6 +53,10 @@
                                         <img src="assets/images/items/Trade_stone.png">
                                         <div data-bind="text: player._itemList['Trade_stone']()"></div>
                                     </div>
+									<div class=col style="cursor: pointer" data-bind="css:{ 'item-selected': ItemHandler.stoneSelected() === 'Sun_stone'}, click: function(){ ItemHandler.stoneSelected('Sun_stone'); ItemHandler.pokemonSelected('')}">
+									<img src=assets/images/items/Sun_stone_small.png>
```

Comment:
Could you put the source in quote marks please
```suggestion
                                        <img src="assets/images/items/Sun_stone_small.png">
```

Reply:
Sun_Stone is a 90x90px asset used in the underground, which is way to big for evo stone display 
Sun_Stone_Small is the same asset, only 32x32px

<img> tag is not closed anywhere else in the evo display and acc. [stackoverflow](https://stackoverflow.com/questions/23890716/why-is-the-img-tag-not-closed-in-html/23890817) doesent need to be

Reply:
Ah sorry, missed that a file had been uploaded with the PR :sweat_smile: 

Could you just add the quote marks around the `src` for the `img` and the `class` in the `div` above, and close the `div` below.